test/visibility/except/operatorsExceptions/exceptNegation.compopts

```diff
@@ -0,0 +1 @@
+DefinesOp.chpl --main-module exceptNegation
```

Comment:
I think you could make a directory wide compopts with just DefinesOp, right? And you wouldn't need a --main-module if each test has a `main`?

Reply:
Oh, yeah, good point!